packages/ui/src/app/pages/Council/Council.tsx

```diff
@@ -41,7 +41,7 @@ export const Council = () => {
 
   const isCouncilorLoading = !isRefetched && (isLoading || isLoadingCouncilors)
 
-  const rewardPerDay = useMemo(() => reward?.period?.mul(reward?.amount ?? asBN(0)) ?? asBN(0), [reward])
+  const rewardPerDay = useMemo(() => reward?.period?.mul(reward?.singleCouncilorAmount ?? asBN(0)) ?? asBN(0), [reward])
   const main = (
     <MainPanel>
       <StatisticsStyle>
```

Comment:
To make things 100% clear could you rename "Councilor Reward" to "Per Councilor Reward" please

Reply:
You mean in the UI here?
![Uploading CleanShot 2023-12-22 at 12.00.15@2x.png…]()

Reply:
Yes

Reply:
Changed to `Single Councillor Reward`